spec/acceptance/qpid_spec.rb

```diff
@@ -47,5 +47,9 @@ class { 'qpid':
     describe package('qpid-tools') do
       it { is_expected.not_to be_installed }
     end
+
+    describe file('/var/lib/qpidd') do
+      it { is_expected.not_to exist }
```

Comment:
NIT: as long as we are testing that the directory removal works, we might as well also test that we properly added it in the first place. Under the context 'with default parameters' above, you could add a similar test:

```
describe file('/var/lib/qpidd') do
  it { is_expected.to be_directory }
end
```